tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -338,6 +410,31 @@ async def test_reborn_v2_text_turn_persists(reborn_v2_server):
         )
 
 
+async def test_reborn_v2_disabled_tool_does_not_route_through_shell(
+    reborn_v2_server, disabled_echo_shell_ask_policy
```

Ruff (0.15.18)

[warning] Unused function argument: `disabled_echo_shell_ask_policy`

(ARG001)